deploy/charts/rook-ceph/values.yaml

```diff
@@ -133,7 +133,7 @@ csi:
   allowUnsupportedVersion: false
 
   # CephCSI RBD plugin Volumes
-  # csiRBDPluginVolumes: |
+  # csiRBDPluginVolume:
```

Comment:
Change here is not required as it's an array of values.

Reply:
No, it's required. Uncommenting this section, as the docs say, makes the operator fail. Check logs in pr description.

Reply:
The resulting configmap requires a multi-line string, so this example must have just been following the [operator.yaml](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L144-L151) example that requires the `|` for multi-line strings. So helm created the multi-line string in the configmap as expected with your change?

Reply:
When I added the values as a string (just like it was here, with the `|`), the operator couldn't parse the value. It yielded these error logs:

```
2022-11-04 09:54:47.489851 I | op-k8sutil: CSI_RBD_PLUGIN_VOLUME="|\n - name: lib-modules\n hostPath:\n path: /run/current-system/kernel-modules/lib/modules/\n - name: host-nix\n hostPath:\n path: /nix" (configmap)
2022-11-04 09:54:47.489907 W | ceph-csi: failed to parse "|\n - name: lib-modules\n hostPath:\n path: /run/current-system/kernel-modules/lib/modules/\n - name: host-nix\n hostPath:\n path: /nix" for "CSI_RBD_PLUGIN_VOLUME". json: cannot unmarshal string into Go value of type []v1.Volume
2022-11-04 09:54:47.489916 I | op-k8sutil: CSI_RBD_PLUGIN_VOLUME_MOUNT="|\n - name: host-nix\n mountPath: /nix\n readOnly: true" (configmap)
2022-11-04 09:54:47.489937 W | ceph-csi: failed to parse "|\n - name: host-nix\n mountPath: /nix\n readOnly: true" for "CSI_RBD_PLUGIN_VOLUME_MOUNT". json: cannot unmarshal string into Go value of type []v1.VolumeMount
```

Then I went and removed the `|`, redeployed, and all started working as it should.

So yes, according to my experience, this change is a requirement.